Dockerfile

```diff
@@ -15,31 +11,20 @@ RUN wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-k
 # We add postgis as well to prevent build errors (that we dont see on local builds)
 # on docker hub e.g.
 # The following packages have unmet dependencies:
-RUN apt-get -y update; apt-get install -y postgresql-client-9.6 postgresql-common postgresql-9.6 postgresql-9.6-postgis-2.4 postgresql-9.6-pgrouting netcat
+RUN apt -y update; apt install -y postgresql-client-9.6 postgresql-common postgresql-9.6 postgresql-9.6-postgis-2.4 postgresql-9.6-pgrouting netcat
 
 # Open port 5432 so linked containers can see them
 EXPOSE 5432
 
-# Run any additional tasks here that are too tedious to put in
-# this dockerfile directly.
-ADD env-data.sh /env-data.sh
-ADD setup.sh /setup.sh
-RUN chmod +x /setup.sh
-RUN /setup.sh
-
 # We will run any commands in this when the container starts
-ADD docker-entrypoint.sh /docker-entrypoint.sh
+ADD env-data.sh /
+ADD docker-entrypoint.sh /
 ADD setup-conf.sh /
 ADD setup-database.sh /
 ADD setup-pg_hba.sh /
 ADD setup-replication.sh /
 ADD setup-ssl.sh /
 ADD setup-user.sh /
-ADD postgresql.conf /tmp/postgresql.conf
-RUN chmod +x /docker-entrypoint.sh
-
-# Optimise postgresql
-RUN echo "kernel.shmmax=543252480" >> /etc/sysctl.conf
-RUN echo "kernel.shmall=2097152" >> /etc/sysctl.conf
+RUN chmod +x /*.sh
```

Comment:
Should we not be running setup.sh here. All other scripts are referenced in  entrypoint